dash/frontend/src/app/modules/public/pages/login/login.component.html

```diff
@@ -35,7 +35,7 @@
             <input id="password" matInput type="password" formControlName="password">
           </mat-form-field>
           <div class="form-action-button" *ngIf="selectedInSiteCredentialAuthenticationMethod?.type === 'LOCAL_AUTH' || selectedInSiteCredentialAuthenticationMethod?.type === 'LDAP'">
-            <button id="submit'" mat-raised-button type="submit">Login</button>
+            <button id="submit'" mat-raised-button mat-button type="submit">Login</button>
```

Comment:
Can we just make this a <mat-button> instead of a <button>?

Reply:
And we can probably set color="primary" here

Reply:
Material design doesn't like having mat-raised-button and mat-button on the same element, you likely just want to set color="primary" as Becky suggested, and remove the plain mat-button directive.

Reply:
It used to be grey I thought. If blue works, that's fine for me.

Reply:
Yes, we want it to be the primary color for consistency across the platform :)